.github/workflows/code-quality.yml

```diff
@@ -7,6 +7,20 @@ on:
       - "*.latest"
   pull_request:
   workflow_dispatch:
+    inputs:
+      dbt_adapters_branch:
+        inputs:
+          description: "The branch of dbt-adapters to evaluate"
+          type: string
+          required: true
```

Comment:
I don't think you can do both `required` and `default`. And either way, you don't need `required` since you're specifying a `default`. I'm commenting here, but this comment applies to all of the inputs in this PR.

Reply:
You technically can do it but it isn't necessary